app/js/components/names.jsx

```diff
@@ -134,7 +134,12 @@ class Name extends React.Component {
     }
     return (
       <div>
-        <form className='form-horizontal'>
+          <h3 className=''
+            name=''
+          > {this.state.givenName} {' '} {this.state.middleName } {''} {this.state.familyName}</h3>
+          <a> Edit </a>
+
+        {/*<form className='form-horizontal'>
```

Comment:
Did you also look at our conventions in regards to commented out code? https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-Donotcommentoutcode